### PR TITLE
51 make docker image for asterisk

### PR DESCRIPTION
**Pull request by tsearle**
make docker image for asterisk & setup the minikube environment

### CODE REVIEW

File: .github/workflows/asterisk.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
```
Comment (gabriel0003) [inline .github/workflows/asterisk.yml:1]: Can you maybe rename this file to build-asterisk?

**Review by gabriel0003 — DISMISSED** [review]

**Review by gabriel0003 — APPROVED** [review]

File: deployment/k8s/local-minikube/apps-config/kamailio-k8s-service.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Service                    # Type of kubernetes resource
+metadata:
+  name: kamailio-service   # Name of the resource
+  annotations:
+    'consul.hashicorp.com/service-sync': 'true'
```
Comment (edifirut) [inline deployment/k8s/local-minikube/apps-config/kamailio-k8s-service.yaml:6]: this line needs to be removed

File: deployment/k8s/local-minikube/apps-config/kamailio-k8s-service.yaml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
```
Comment (edifirut) [inline deployment/k8s/local-minikube/apps-config/kamailio-k8s-service.yaml:1]: please remove the comments. not relevant

File: deployment/k8s/local-minikube/postgres/postgresql-deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    matchLabels:
+      app: oasis-postgres
+  template:
+    metadata:
+      labels:
+        app: oasis-postgres
+    spec:
+      containers:
+        - name: oasis-postgres
+          image: postgres:10.1
```
Comment (edifirut) [inline deployment/k8s/local-minikube/postgres/postgresql-deployment.yaml:17]: please use the 13.4 version, as the AWS environments

File: deployment/k8s/local-minikube/postgres/postgresql-storage.yaml [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    type: local
+    app: postgres
+spec:
+  storageClassName: manual
+  capacity:
+    storage: 5Gi
+  accessModes:
+    - ReadWriteMany
+  hostPath:
+    path: "/mnt/data"
```
Comment (edifirut) [inline deployment/k8s/local-minikube/postgres/postgresql-storage.yaml:15]: do we need to add a postfix like /postgres ? maybe we are going to have more volumes and we should map them under the service name in /mnt/data

**Review by edifirut — APPROVED** [review]

**Pull request merged**